From: Thierry Reding <treding@nvidia.com>
Date: Thu, 18 Feb 2016 14:32:10 +0100
Subject: PCI: mvebu: Restrict build to 32-bit ARM
Git-commit: 61d9e854dfb91b46b99ffd3d31fc7d45d2b95f31
Patch-mainline: v4.5-rc6
References: fate#319899 fate#320700

This driver uses PCI glue that is only available on 32-bit ARM.  This used
to work fine as long as ARCH_MVEBU and ARCH_DOVE were exclusively 32-bit,
but there's a patch in the pipe to make ARCH_MVEBU also available on 64-bit
ARM.

[bhelgaas: changelog; patch is coming but not merged yet]
Signed-off-by: Thierry Reding <treding@nvidia.com>
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Acked-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/pci/host/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/pci/host/Kconfig
+++ b/drivers/pci/host/Kconfig
@@ -14,6 +14,7 @@ config PCI_DRA7XX
 config PCI_MVEBU
 	bool "Marvell EBU PCIe controller"
 	depends on ARCH_MVEBU || ARCH_DOVE
+	depends on ARM
 	depends on OF
 
 config PCIE_XILINX_NWL
